assets/components/paymentFailureMessage/paymentFailureMessage.scss

```diff
@@ -0,0 +1,42 @@
+.component-payment-failure-message {
+  padding: ($gu-v-spacing / 2) $gu-h-spacing $gu-v-spacing;
+  color: gu-colour(news-garnett-main-1);
+  background-color: gu-colour(news-faded);
+  font-family: $gu-sans-web;
+  margin-bottom: 20px;
+
+  @include mq($from: tablet) {
+    width: 404px;
+  }
+
+}
+
+.svg-exclamation-alternate {
+  display: inline-block;
+  margin-right: 8px;
+  vertical-align: bottom;
+}
+
+.svg-exclamation-alternate__circle {
+  fill: gu-colour(news-garnett-main-1);
+}
+
+.svg-exclamation-alternate__exclamation-mark {
+  fill: gu-colour(garnett-neutral-5)
+}
+
+.component-payment-failure-message__error-message {
+  vertical-align: bottom;
+  display: inline-block;
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.component-payment-failure-message__small-print {
```

Comment:
Should this be in `$gu-text-sans-web`?

Reply:
I assume this is rhetorical (I have no idea) - so 👍